Added admonition for k8s delegate (#7374)
harness/developer-hub: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md

Local runner build infrastructure is recommended for small, limited builds, such as a one-off build on your local machine. Consider [other build infrastructure options](/docs/category/set-up-build-infrastructure) for builds-at-scale.

:::note

> You can also use a [Harness Kubernetes Delegate](/docs/platform/delegates/install-delegates/overview#install-the-default-delegate) instead of a Harness Docker Delegate.

Reviewer: ```suggestion
You can also use a [Harness Kubernetes delegate](/docs/platform/delegates/install-delegates/overview#install-the-default-delegate) instead of a Harness Docker delegate.
```

Author: So I capitalized delegate in this context because Harness precedes/modifies it, and the style guide recommends capitalizing the delegate in that case

Reviewer: @SushrutHarness we've normally capitalized delegate when Harness immediately proceeds it. I updated my comment to remove "Harness". Up to you how you implement it.

> It is possible to use a [Harness Kubernetes Delegate](/docs/platform/delegates/install-delegates/overview#install-the-default-delegate) instead of a Harness Docker Delegate.

Author: ```suggestion
You can also use a [Harness Kubernetes Delegate](/docs/platform/delegates/install-delegates/overview#install-the-default-delegate) instead of a Harness Docker Delegate.
```
